cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelper.java

```diff
@@ -48,7 +48,7 @@ class ReplicationThrottleHelper {
   static final String LEADER_THROTTLED_REPLICAS = getLogConfig(LogConfig.LEADER_REPLICATION_THROTTLED_REPLICAS_CONFIG);
   static final String FOLLOWER_THROTTLED_REPLICAS = getLogConfig(LogConfig.FOLLOWER_REPLICATION_THROTTLED_REPLICAS_CONFIG);
   public static final long CLIENT_REQUEST_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(30);
-  static final int RETRIES = 30;
+  static final int RETRIES = 5;
```

Comment:
could we maybe instead reduce/remove/limit the [exponential growth](https://github.com/DataDog/cruise-control/blob/upstream-main/cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsUtils.java#L31)? the requests are already low throughput so just retrying every few seconds for a while should be reasonable IMO. That way we get faster feedback since even with 5 retries we do `5, 10, 20, 40, 80` second sleeps

Reply:
I had this ready with an upper bound, but I was wondering how much trickier that would be to get merged upstream, so I aborted it.
Can add that again, sure. It's used in a couple other places, I'm gonna make sure it doesn't affect those.